RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -1075,8 +1102,9 @@ boolean loadConsumables(string _type, ConsumeAction[int] actions)
 		{
 			int n = count(actions);
 			actions[n] = MakeConsumeAction(it);
-			if (obtain_mode == AUTO_OBTAIN_PULL)
+			if (obtain_mode == AUTO_OBTAIN_PULL && !in_small())
```

Comment:
there was some discussion in discord about this could lead to pulling many small consumables. Agree we should be less conservative with our pulls. Will leave to a future PR